docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -702,3 +708,12 @@ function layout_builder_custom_preprocess_media(&$variables) {
     ] + $variables['attributes'];
   }
 }
+
+/**
+ * Implements hook_views_plugins_display_alter().
+ */
+function layout_builder_custom_views_plugins_display_alter(&$displays) {
+  if (!empty($displays['block']['class']) && $displays['block']['class'] == Block::class) {
+    $displays['block']['class'] = ListBlock::class;
```

Comment:
>Check other views allowed to be placed by webmaster because the views label is now hidden and replaced by the headline field.

It was a test instruction. The ones I tested could be abused (and addressed in follow-ups) but nothing was "broken" that I saw. I will do another check while making the other change.

Reply:
No worries. I just wanted to double check since I hadn't been entirely sure about this as the "right" approach when I was working on it initially.